godtools/App/Flows/ToolSettings/Observer/Protocols/LinkShareable.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LinkShareable.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 8/20/24.
+//  Copyright © 2024 Cru. All rights reserved.
+//
+
+import Foundation
+
+protocol LinkShareable {
```

Comment:
Hey @rachaelblue these changes are looking good.  The View fix looks good too.  

I'm wondering if we should think about adding the dependencies needed into the protocols?  Going to think about that a bit.

Reply:
This isn't me reviewing the PR haha, but @rachaelblue I really like how you set up Tool Options, allowing easy disable and enable of the different parts.

Reply:
Hey @aaronlaib welcome.  I agree.

Reply:
Sounds good @levieggertcru.  And thanks @aaronlaib!